src/benchmarks/eo/class_simple.c

```diff
@@ -7,8 +7,8 @@
 
 #define MY_CLASS SIMPLE_CLASS
 
-EOAPI const Efl_Event_Description _SIMPLE_FOO = EFL_EVENT_DESCRIPTION("foo");
-EOAPI const Efl_Event_Description _SIMPLE_BAR = EFL_EVENT_DESCRIPTION("bar");
+const Efl_Event_Description _SIMPLE_FOO = EFL_EVENT_DESCRIPTION("foo");
```

Comment:
Why not EO_API?

Reply:
Por que class_simple.c é um executável, não uma biblioteca. Ele não precisa exportar ou importar o simbolo que ele mesmo define e usa.